docs/content/preview/integrations/spring-framework/sdyb.md

```diff
@@ -48,8 +48,256 @@ The project definition includes the following dependencies:
 
 ## Examples
 
+### 1. Build a Java application
 
-The following example demonstrates how to create a basic shopping cart using SDYB:
+The following tutorial illustrates how to build a Java application with Spring Data YugabyteDB and use the YSQL API to connect to and interact with YugabyteDB.
```

Comment:
what is it, and what does it do?

Reply:
We are already giving an overview of spring data at the beginning. This example is just a build an app with YB. So I guess the existing details are good.